docs/api.md

```diff
@@ -72,26 +72,31 @@ in more detail in the [API Changes documentation](https://github.com/kubernetes/
 
 ## API groups
 
-To make it easier to extend the Kubernetes API, we are in the process of implementing [*API
-groups*](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/proposals/api-group.md).  These are simply different interfaces to read and/or modify the
-same underlying resources.  The API group is specified in a REST path and in the `apiVersion` field
-of a serialized object.
+To make it easier to extend the Kubernetes API, we implemented [*API groups*](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/proposals/api-group.md).
+These are simply different interfaces to read and/or modify the same underlying resources.
```

Comment:
Remove this line, it may have been right in some context (we moved a couple resources) but it's totally wrong here :)

If you want, replace with something like "API Groups are a logical division of the resources in the API, allowing us to evolve different resources at different rates".